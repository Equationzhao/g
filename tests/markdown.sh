output="$(g -term-width 200 --no-config --no-update --icons --permission --size --markdown tests/test_data)"
echo "$output" | diff - tests/markdown.stdout
