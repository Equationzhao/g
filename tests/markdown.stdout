# path: /Users/equationzhao/g/tests/test_data
| Permissions | Size | Name |
| --- | --- | --- |
| [1md[1mr[1mw[1mx[1mr-[1mx[1mr-[1mx[0m  | 128.0 B[0m |  ?.dir[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | 21.0 B[0m |  '   space file '[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  0.1.112[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  0.1.13[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  0.1.2[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  0.2.10[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  0.2.2[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  0.3.0[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  0.3.84[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  1.0.1[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  1.4.1[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | 5.0 B[0m |  1.txt[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | 7.0 B[0m |  10.json[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | 100.0 KiB[0m |  100KiB[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | 6.0 B[0m |  11.xml[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | 6.0 B[0m |  12.yml[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | 5.0 B[0m |  13.md[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  14.gif[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  2.19.1[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  2.9.1[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | 3.0 B[0m |  2.c[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | 5.0 B[0m |  3.cpp[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  4.py[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | 6.0 B[0m |  5.java[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  6.sh[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | 6.0 B[0m |  7.html[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | 5.0 B[0m |  8.css[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | 4.0 B[0m |  9.js[0m |
| -[1mr[1mw-[1mr--[1mr--[0m  | -[0m |  9.js~[0m |
| [1ml[1mr[1mw[1mx[1mr-[1mx[1mr-[1mx[0m  | 5.0 B[0m |  1.txt.link[0m => [0m/Users/equationzhao/g/tests/test_data/1.txt[0m |
